tox.ini

```diff
@@ -1,12 +1,14 @@
 [tox]
-envlist = cov-clean,py{39,36}-noredis,py{39,38,37,36},cov-report
+envlist = cov-clean,py{39,36}-noredis,py37-withendpoint,py{39,38,37,36},cov-report
```

Comment:
The choice of 3.7 as the version for this is admittedly a little random. Given that we have services using 3.7 it seemed a good guess as to a python which everyone might have installed.

It doesn't matter too much, since the factor allows usages like `tox -e py36-withendpoint` to call these tests.